modules/apps/frontend-js/frontend-js-spa-web/test/senna/app/App.js

```diff
@@ -1461,7 +1445,7 @@ describe('App', function () {
 		jest.spyOn(
 			utils,
 			'getCurrentBrowserPathWithoutHash'
-		).mockImplementation(() => '/path');
+		).mockImplementation(() => '/asdasdasdasas');
```

Comment:
Well... this is embarrasing... 😂 

Removing this mock will make tests fail... but as can be seen in the change, it doesn't matter what the implementation does 🤷